examples/specs/window_transform_time_spent_in_class.vl.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Following the discussion here: https://github.com/vega/vega-lite/pull/3199#discussion_r175275403

I'm still not convinced that this is a good example.  

If you can explain a specific analysis question that this plot can answer well, I can be more convinced. (Why would you want to stack the time spent in school for the two students?)

Reply:
Dropped this example.